apps/web/src/stores/editor-store.ts

```diff
@@ -1163,12 +1163,17 @@ class EditorStore extends BaseStore<EditorStore> {
     this.updateSession(id, ["default", "new"], { saveState: saveState });
   };
 
-  toggleProperties = (toggleState?: boolean) => {
+  toggleProperties = (
+    toggleState?: boolean,
+    toggleTableOfContents?: boolean
+  ) => {
     this.set((state) => {
       state.arePropertiesVisible =
         toggleState !== undefined ? toggleState : !state.arePropertiesVisible;
     });
-    this.toggleTableOfContents(false);
+    if (toggleTableOfContents !== false) {
+      this.toggleTableOfContents(false);
+    }
```

Comment:
In what case we want to toggle table of contents with properties? I think this can be removed as it's not needed anymore. There is a dedicated function to toggle TOC already which should be used.

Reply:
Whenever properties is toggled, it closes the table of contents, this ensures that only one pane is open at a time

Reply:
Let's keep the existing behavior and:

1. If you open TOC, it should close properties if already opened
2. If you close properties, do nothing with the TOC.
3. If you open properties, do nothing with the TOC.

From what I can tell, it is a bug in the beta app because the stable version behaves correctly. The TOC never closes unless you close it by clicking X.

Reply:
Just trying the beta app again, it seems that the UI has changed quite a bit and now properteis and toc are part of the editor, not floating on top. Both show in the same container I think?

Reply:
They are separate panes

Reply:
@ammarahm-ed do let me know what needs to be done